src/lib/components/CommunitySupportChat.svelte

```diff
@@ -87,9 +91,51 @@
 </ul>
 
 <style>
+    .web-user-box-image {
```

Comment:
why do we need this many props with css when the `style="..."` already has the similar specs? Can you confirm which do we want here.